packages/contracts-bedrock/scripts/slither.sh

```diff
@@ -4,8 +4,10 @@ rm -rf artifacts forge-artifacts
 
 # See slither.config.json for slither settings
 if [[ -z "$TRIAGE_MODE" ]]; then
+  echo "Building contracts"
+  forge build --build-info --force
```

Comment:
Should this also be run in triage mode?

Reply:
This fix is primarily to make sure that slither doesn't time out in CI due to lack of output - triage mode is normally ran locally, so we shouldn't have to do this there.